File: Python/Bybit/REST_Ticker.py
from pybit.unified_trading import HTTP
import get_symbols

symbol = get_symbols.get_symbols()

session = HTTP(testnet=False)

print(len(symbol))

# for i in range(len(symbol)):
    
#     try:
#         print(session.get_tickers(
#             category="spot",
#             symbol=symbol[i],
#         ))

#     except:
#         continue

File: GO/Bybit/REST_Ticker.go
package main

import (
  "fmt"
  "net/http"
  "io/ioutil"
  "regexp"
)

func main() {

  url := "https://api.bybit.com/v5/market/tickers?category=spot"
  method := "GET"

  client := &http.Client {
  }
  req, err := http.NewRequest(method, url, nil)

  if err != nil {
    fmt.Println(err)
    return
  }
  res, err := client.Do(req)
  if err != nil {
    fmt.Println(err)
    return
  }
  defer res.Body.Close()

  body, err := ioutil.ReadAll(res.Body)
  if err != nil {
    fmt.Println(err)
    return
  }
  fmt.Println(string(body)[0])

  re := regexp.MustCompile(`"symbol":\"(.*?)\",\"bid1Price`)
  fmt.Printf("%q\n", re.FindAllStringSubmatch(string(body), -1))
}